wiki/Getting-Started-UIKit.md

```diff
@@ -6,6 +6,10 @@ This guide will walk you through getting a [Workflow](https://wwt.github.io/Swif
 
 For instructions on SPM and CocoaPods, [check out our installation page.](https://github.com/wwt/SwiftCurrent/wiki/Installation#swift-package-manager)
 
+## IMPORTANT NOTE
```

Comment:
The comments help and are a step in the right direction to reduce confusion. 

Given the wall of code particularly in building programmatic view layouts the comments still run the risk of being lost. While not resulting in a nice layout removing some of the additional view code and/or splitting out making, customizing, and laying out the views from what's needed for SwiftCurrent.

Reply:
My hope is that if people seem lost in what where the SwiftCurrent code is, they will do a search for `// SwiftCurrent`.

I'm also hopeful that just calling out that we have marked the SwiftCurrent lines, that they will process the rest of the code as basic app development instead of being important to learning the library.